### PR TITLE
debugs and new function 

### PR DESCRIPTION
**Pull request by m-curti**
Debugged the 'angle_to_sexigesimal' function within the buggy_function script
Included 'is_prime' function in the simple_functions script

### CODE REVIEW

**Review by shilpi1958 — COMMENTED** [review]
Hi,
I would like to review this 
:)

cheers,
Shilpi

File: Curti/template/simple_functions.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+	if a < 2 : 
+		return True
+
+	else : 
+		for i in range(2,a) : 
+
+			if not (a%i) : 
+				return False
+		else :
```
Comment (shilpi1958) [inline Curti/template/simple_functions.py]: The last else has no preceding if statement. 
You could remove else statement and the function will work just fine.
```suggestion
	
```

File: Curti/template/simple_functions.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	if a < 2 : 
+		return True
+
+	else : 
+		for i in range(2,a) : 
+
+			if not (a%i) : 
+				return False
+		else : 
+			return True
```
Comment (shilpi1958) [inline Curti/template/simple_functions.py]:
```suggestion
		return True		
```

File: Curti/template/simple_functions.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+def factorial(value):
+    if value == 0:
+        return 1
+    else:
+        return value * factorial(value - 1)
+
+
+def is_prime(a) : 
+
```
Comment (shilpi1958) [inline Curti/template/simple_functions.py:16]: what if a is negative? your function will still return true

Comment (m-curti) [reply Curti/template/simple_functions.py:16]: should be ok now.
Thank you!

Comment (shilpi1958) [reply Curti/template/simple_functions.py:16]: :)

**Comment by astrofrog** [comment]
Thanks for opening the pull request and for the review(s)! Below I have provided a few comments about the *process* of the pull request and review. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. Also, because there are pending suggestions, we will not merge this PR until the end of Astro Hack Week.

* @Utty87 - the use of 'debugs' in the title could probably be phrased more explicitly as 'Fixed bugs in ... and'. Also, if you push new commits, it would be good to indicate in a comment whether this is ready for another review, and whether you have addressed all the requested changes.

* @shilpi1958 - good review, and nice use of suggested changes, but you did not "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments. In this case, it seems like 'Request changes' would have been appropriate.

**Comment by bsipocz** [comment]
As a closure to AstroHackWeek, we go ahead and merge this now. Thank you both.

**Pull request merged**